.github/workflows/pr-review-companion.yml

```
@@ -66,7 +66,7 @@ jobs:
         id: setup-python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.11.0"
```

Or is this better?

```suggestion
          python-version: "3.11"
```

---

The best would be to use the same [version as yari](https://github.com/mdn/yari/blob/f6252a5f708adc161a3cf7fd1545d7d21f57f689/.github/workflows/prod-build.yml#L101):

```suggestion
          python-version: "3.10"
```

---

Great, applied the suggestion, tnx